vcs/gitcmd/repo.go

```diff
@@ -614,6 +628,20 @@ func (r *Repository) UpdateEverything(opt vcs.RemoteOpts) error {
 		cmd.Env = []string{"GIT_SSH=" + gitSSHWrapper}
 	}
 
+	if opt.HTTPS != nil {
```

Comment:
All right, I've looked into it, and I'm not finding a concrete alternative way that is an overall improvement.

Since this PR adds functionality in a consistent way with the existing code, I think it's okay to keep it as is, but we can follow it up with a refactor afterwards.